Sources/Configuration/Providers/EnvironmentVariables/EnvironmentVariablesProvider.swift

```diff
@@ -142,6 +142,20 @@ public struct EnvironmentVariablesProvider: Sendable {
 
         /// A decoder of arrays from a string.
         var arrayDecoder: EnvironmentValueArrayDecoder
+
+        /// A decoder of bool values from a string
+        static func decodeBool(from string: String) -> Bool? {
+            let stringLowercased = string.lowercased()
+            return if ["true", "false"].contains(stringLowercased) {
+                stringLowercased == "true"
+            } else if ["yes", "no"].contains(stringLowercased) {
+                stringLowercased == "yes"
+            } else if ["1", "0"].contains(stringLowercased) {
+                stringLowercased == "1"
+            } else {
+                nil
+            }
```

Comment:
```suggestion
            switch string.lowercased() {
            case "true", "yes", "1": true
            case "false", "no", "0": false
            default: nil
            }
```

Just a bit shorter :)